lib/fittings/json_error_handler.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var util = require('util');
+var http = require('http');
+var injector = require('../../index').injector;
+
+module.exports = function create(fittingDef, bagpipes) {
```

Comment:
'bagpipes' is defined but never used.<br>'fittingDef' is defined but never used.